contracts/starknet/src/entry/structs.cairo

```diff
@@ -61,6 +61,19 @@ struct Currency {
     ethereum_address: felt,  // optional
 }
 
+struct FuturesCurrency {
```

Comment:
Why not just use Currency ?

Reply:
I decided to create two separate entities, because the prices displayed in futures and in spot can vary for the same currency

Reply:
Right but the currency remains the same let's say for the BTC/USD Perp Contract, currencies are BTC and USD

Reply:
Oh I see, you are right !